packages/graphql-api/src/modules/accounts/schema/mutation.ts

```diff
@@ -10,5 +10,6 @@ export default (config: AccountsModuleConfig) => gql`
     # Example: Login with password
     # authenticate(serviceName: "password", params: {password: "<pw>", user: {email: "<email>"}})
     authenticate(serviceName: String!, params: AuthenticateParamsInput!): LoginResult
+    authenticateWithoutSessionCreation(serviceName: String!, params: AuthenticateParamsInput!): Boolean
```

Comment:
`noSession` could be add as an optional argument to the `authenticate` mutation if you want to avoid creating this extra `authenticateWithoutSessionCreation` names

Reply:
I thought about it but it will make the `loginWithService` function more complex as it would return `LoginResult | boolean`.